gradle.properties

```diff
@@ -2,7 +2,7 @@ org.gradle.caching=true
 puppycrawlCheckstyleVersion=8.18
 group=org.ballerinalang
 version=1.0.10-SNAPSHOT
-ballerinaLangVersion=2.0.0-alpha7-SNAPSHOT
+ballerinaLangVersion=2.0.0-alpha7-20210326-211300-6ccfb47f
```

Comment:
This needs to be updated, to include lang changes of https://github.com/ballerina-platform/ballerina-lang/pull/29659